go.mod

```diff
@@ -57,7 +57,7 @@ require (
 	github.com/devigned/tab v0.1.2-0.20190607222403-0c15cf42f9a2
 	github.com/dgraph-io/badger/v3 v3.2103.1
 	github.com/digitalocean/go-libvirt v0.0.0-20240709142323-d8406205c752
-	github.com/docker/docker v26.1.5+incompatible
+	github.com/docker/docker v27.0.3+incompatible
```

Comment:
From https://github.com/moby/moby/security/advisories/GHSA-v23v-6jw2-98fq this has a vulnerability we need to avoid, think we'd need to go to v27.1.1+ to avoid it as there's no v27.0.4 for whatever reason.

Reply:
It doesn't actually affect us but the CVSS is 10/10 so we do not want it showing up in CVE scans of Beats regardless.

Reply:
Dependabot only auto-updates for github.com/elastic dependencies, for non-elastic dependencies it only suggests updates to remove CVEs.

We could turn on dependabot for everything, but that'd be a lot of PRs to keep up with.

Reply:
@cmacknz I didn't know that thanks for the explanation. It might be a lot, wonder if we could just get it to run like once a month with all deps in a single PR.

@pazone ^ would that be possible?